docs/source-pytorch/advanced/pruning_quantization.rst

```diff
@@ -113,3 +117,9 @@ You can further customize the callback:
 
     batch = iter(my_dataloader()).next()
     qmodel(batch[0])
+
+
+Post-training Quantization
+==========================
+
+If you want to quantize a fine-tuned model with PTQ, it is recommended to adopt a third party API names Intel® Neural Compressor, read more `here <https://intel.github.io/neural-compressor>`_, which provides a convenient tool for accelerating the model inference speed on Intel CPUs and GPUs.
```

Comment:
I think here in this paragraph here we could easily link to the main document `advanced/post_training_quantization`. What do you think?

Reply:
Thanks @awaelchli. May I know how to add to the right link to "advanced/post_training_quantization" when the doc being compiled?

Reply:
Hi, Adrian, this link problem has been fixed in the update.